symbols/arm9.yml

```
@@ -9343,6 +9343,71 @@ arm9:
         
         r0: matrix_4x3 pointer
         r1: GXFIFO pointer
+    - name: IncrementThreadCount
+      address:
+        EU: 0x20794E8
+        NA: 0x2079150
+      description: |-
+        Increments thread_info::thread_count by 1 and returns the new value.
+        
+        return: thread_info::thread_count + 1
+    - name: InsertThreadOnList
+      address:
+        EU: 0x2079630
+        NA: 0x2079298
+      description: |-
+        Inserts a new thread on the linked thread list (see thread_info::first_thread).
+        
+        The order of the list will be maintained.
```

I think this could be misinterpreted as the new thread being inserted at the end, thereby "maintaining" the old list order. It might be clearer to phrase it like:

```suggestion
        The thread is inserted in sorted order.
```

---

Oh, you're right. I didn't think of that. I'll change it.